charts/cluster-scanner/values.yaml

```diff
@@ -0,0 +1,124 @@
+#basic Default values for cluster-scanner.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  sysdigHost: ""
+  sysdigAccessKey: ""
+  clusterName: ""
+  sysdigSkipVerifyCertificate: false
+
+  scannerMode: "local" # or "multi"
+  loggingLevel: "INFO"
```

Comment:
Can we try to keep the same structure we have in the main chart here?

https://github.com/sysdiglabs/charts/blob/master/charts/sysdig-deploy/values.yaml#L1-L17

Reply:
@mavimo I'll do, but when using `custom` reagion, which field is supposed to specify the host/baseURL for the Sysdig backend?

Reply:
We have updated it see: https://github.com/sysdiglabs/charts/blob/f3c19117aae903614c4d7aa59a8848db1ca81f06/charts/cluster-scanner/values.yaml#L5-L26